src/plot/jim.jl

```diff
@@ -6,257 +6,9 @@ jiffy image display
-	elseif fft0
-		z = FFTView(z)[x,y]
-	end
-
-	if minimum(z) ≈ maximum(z) # uniform or nearly uniform image
-		x = 1:size(z,1)
-		y = 1:size(z,2)
-		plot(aspect_ratio=aspect_ratio,
-			xlim = [x[1], x[end]],
-			ylim = [y[1], y[end]],
-			title=title,
-			yflip=yflip,
-			xlabel=xlabel,
-			ylabel=ylabel,
-			xtick=xtick,
-			ytick=ytick,
-		)
-		tmp = (minimum(z) == maximum(z)) ? "Uniform $(z[1])" :
-			"Nearly uniform $((minimum(z),maximum(z)))"
-		annotate!((sum(x)/length(x), sum(y)/length(y), tmp, :red))
-	else
-		heatmap(xy..., z', transpose=false,
-			aspect_ratio=aspect_ratio,
-			clim=clim,
-			color=color,
-			colorbar=colorbar,
-			title=title,
-			yflip=yflip,
-			xlabel=xlabel,
-			ylabel=ylabel,
-			xtick=xtick,
-			ytick=ytick,
-		)
-	end
-
-	if n3 > 1 && line3plot # lines around each subimage
-		m1 = (1+size(z,1)) / n1 # add one because of mosaicview non-edge
-		m2 = (1+size(z,2)) / n2
-		plot_box! = (ox, oy) -> plot!(
-			ox .+ [0,1,1,0,0] * n1 .+ 0.0,
-			oy .+ [0,0,1,1,0] * n2 .+ 0.0,
-			line=jim_def[:line3type], label="")
-		for ii=0:n3-1
-			i1 = mod(ii, m1)
-			i2 = ii ÷ m1
-			plot_box!(i1*n1, i2*n2)
-		end
-	end
-	plot!()
-
-end # jim
-
-
-# handle case of complex image
-function jim(z::AbstractArray{<:Number} ;
-			abswarn::Bool = jim_def[:abswarn],
-			kwargs...
-		)
-
-	if !(eltype(z) <: Real)
-		abswarn && (@warn "magnitude at $(caller_name())")
-		z = abs.(z)
-	end
-	jim(z ; kwargs...)
-end
-
-
-"""
-    jim(z, title ; kwargs...)
-"""
-jim(z::AbstractArray{<:Number}, title::AbstractString ; kwargs...) =
-	jim(z ; title=title, kwargs...)
-
-
-"""
-    jim(x, y, z ; kwargs...)
-"""
-jim(x, y, z ; kwargs...) = jim(z ; x=x, y=y, kwargs...)
-
-
-"""
-    jim(x, y, z, title ; kwargs...)
-"""
-jim(x, y, z, title::AbstractString ; kwargs...) =
-	jim(z ; x=x, y=y, title=title, kwargs...)
-
-
-"""
-    jim()
-return docstring if user calls `jim()` with no arguments
-"""
-function jim()
-	@doc jim
-end
-
-
-#"""
-#`jim(abswarn=false)`
-#"""
-#function jim( ; abswarn::Bool=jim_state_abswarn)
-#	global jim_state_abswarn = abswarn
-#end
-
-"""
-    jim(key, value)
-set default value for one of the keys
-"""
-function jim(key::Symbol, value)
-	global jim_def
-	!haskey(jim_def, key) && throw(ArgumentError("no key $key"))
-	jim_def[key] = value
-end
-
-
-"""
-    jim(:test)
-
-`jim(:keys)` return default keys
-
-`jim(:defs)` return `Dict` of default keys / vals
-
-`jim(:key)` return `Dict[key]` if possible
-"""
-function jim(test::Symbol)
-	global jim_def
-	if test === :keys
-		return sort(collect(keys(jim_def)))
-	end
-	if test === :defs
-		return jim_def
-	end
-	if haskey(jim_def, test)
-		return jim_def[test]
-	end
-
-	throw("symbol $test")
-end
+@deprecate jim(a ; kwargs...) MIRTjim.jim(a ; kwargs...)
+@deprecate jim(a, b ; kwargs...) MIRTjim.jim(a, b ; kwargs...)
+@deprecate jim(a, b, c ; kwargs...) MIRTjim.jim(a, b, c ; kwargs...)
+@deprecate jim(a, b, c, d ; kwargs...) MIRTjim.jim(a, b, c, d ; kwargs...)
```

Comment:
```suggestion
@deprecate jim(args... ; kwargs...) MIRTjim.jim(args...; kwargs...)
```

Reply:
oops, after reading your other reply i was making a commit with that same change at the same time so now github won't let me accept your suggestion.   sorry!